src/org/parosproxy/paros/core/proxy/ProxyServer.java

```diff
@@ -399,13 +408,34 @@ public boolean excludeUrl(URI uri) {
                     if (log.isDebugEnabled()) {
                         log.debug("URL excluded: " + uriString + " Regex: " + p.pattern());
                     }
-                    
+

```

Comment:
Formatting changes (if really necessary) should occur in a separate PR.

https://github.com/zaproxy/zaproxy/wiki/DevGuidelines